procbuild/builder.py

```diff
@@ -15,8 +15,10 @@
                'bibderwalt',    'intro',    'texderwalt']
 
 
-def repo(user='scipy'):
-    return f'https://github.com/{user}/scipy_proceedings.git'
+def clone_url(user='scipy', repo='scipy_proceedings'):
+    # TODO can we replace this with
```

Comment:
Want to resolve this question before merging? The PR looks good!

Reply:
We currently use the same logic to update our own build tools as we do to fetch paper PRs, so we won't have that field available unless we make it a constant somewhere or do some refactoring. Given that this is causing paper build failures now, and we are assigning reviewers tomorrow, I'd like to defer a more thorough look.